proposals/lip-0067.md

```diff
@@ -210,13 +209,11 @@ Secret recovery phrase: `target cancel solution recipe vague faint bomb convince
 ```json
 {
   "crypto": {
-    "version": "1",
     "ciphertext": "866c6f1cab3ef67514bdc54cf0143b8b824ebe7c045efb97707c158c81d313cd1a6399b7aa3002248984d39ea2604b0263fe7bdbd8cb04286a9cbd2d353fc79908daab9af04b2528bf4f06a82d79483c",
     "mac": "a476979ca68fe90f3c96f8a5f3f0a9fe33aef8b091d1169861e44a11a680aae9",
-    "cipher": "aes-256-gcm",
+    "cipher": "aes-128-ctr",
     "cipherparams": {
-      "iv": "da7a74acbf34d20ffd3658f9",
-      "tag": "f4282899ed6cb0193e2981dca0d2ae8e"
+      "iv": "da7a74acbf34d20ffd3658f9"
     },
     "kdf": "argon2id",
     "kdfparams": {
```

Comment:
Please check if we have to change the tag length to account for different length of the input key to the aes algo

Reply:
AES-128-CTR mode doesn't have "tag".